server.go

```diff
@@ -335,7 +335,7 @@ func (s *Server) StartAsSubserver(lndClient lnrpc.LightningClient,
 
 	// Start the macaroon service and let it create its default macaroon in
 	// case it doesn't exist yet.
```

Comment:
nit: change the comment so it reflects that `it creates its default macaroon in case it doesn't exist yet and noMacaroonCreation is true`